docs/DeveloperGuide.md

```diff
@@ -595,6 +601,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   - 4a1. SalesPunch alerts the user that the lead status is the same.  
     Use case resumes at step 2
 
+**Use case: Add tasks for a person**
+
+**MSS**
+
+1.  User adds task to a person
+2.  SalesPunch updates the tasklist of the person in the contact list
+
+    Use case ends.
+
+**Extensions**
+
+- 1a. User is able to add more types of tasks to a person. 
```

Comment:
This doesn't reflect the actual behaviour of the code. From my understanding of your implementation, `addtask` does not prompt check if 'the user is able to add more types of tasks'. There are also no checks for whether a stated task description is empty, or invalid etc. So I think there are no extensions to this use case, unless you wish to add a part about requiring a prefix.